src/common/low_precision_transformations/include/low_precision/propagate_through_precision_preserved.hpp

```diff
@@ -99,11 +99,11 @@ class ngraph::pass::low_precision::PropagateThroughPrecisionPreserved : public n
     }
 
     // TODO: possible duplicate: PropagateToInput::getSourceOutputAttribute
-    std::vector<ov::Any> getParentInputRestrictions(
-        const std::shared_ptr<ngraph::Node> node) {
+    std::vector<ov::Any> getParentInputRestrictions(const std::shared_ptr<ngraph::Node> node,
+        const std::vector<ngraph::element::Type>& defaultPrecisions) {
         std::vector<ov::Any> parentAttributes;
-        auto getInput = [](const std::shared_ptr<ngraph::Node>& node, const size_t index) -> Input<Node> {
-            const auto dequantization = NetworkHelper::getDequantization(node, index);
+        auto getInput = [defaultPrecisions](const std::shared_ptr<ngraph::Node>& node, const size_t index) -> Input<Node> {
```

Comment:
Could you please make all lambda captures "by ref" if copy is not required by lambda.

Reply:
All lambda captures are made by ref. Thank you